tests/system/test_dbapi.py

```diff
@@ -344,6 +344,50 @@ def test_DDL_autocommit(shared_instance, dbapi_database):
             op.result()
 
 
+def test_ddl_executemany(shared_instance, dbapi_database):
+    """Check that DDLs in autocommit mode are immediately executed."""
```

Comment:
Isn't this exactly what we know is not happening? What I mean is:
1. If you call `cur.execute(...)` with DDL statement, the DDL statement is NOT executed directly.
2. It is only executed once you call `execute` or `execute_many` with a non-DDL statement.

Can we also test the behavior when the connection is in `autocommit=false` mode? I'm not saying that the behavior should necessarily be the same, but we should have a test that verifies that it does what we want it to do.

EDIT: Looking further down the test it seems that the connection is in `autocommit=false` mode, as we are calling `commit()`. Would you mind verifying whether this comment is correct, and also make sure that we test the behavior for both `autocommit=true|false`.

Reply:
Sorry for the confusion as I copied the code and forgot updating the comment. 
Ideally there should be 4 tests for execute/executemany and autocommit being True/False combinations
There are no such tests. I added one in this PR so as to verify the issue. Should we add all 4 tests?

Reply:
Yes, I think it would be good to add all 4 tests to this PR. That way, we at least know what the current behavior is. Then we can decide whether we want to change any of them, if we see anything strange.

Reply:
Added tests